spec/factories/assessments.rb

```diff
@@ -55,6 +56,11 @@
       end
     end
 
+    trait :decline_with_no_failure_Reasons do
```

Comment:
```suggestion
    trait :decline do
```

I think this can just be `decline` on its own and if a user wants additional failure reasons they can add them. I'd also suggest putting this next to the `award` trait so they're easier to find together.